tesseract-clickhouse/tests/integration.rs

```diff
@@ -49,29 +32,18 @@ fn test_query() {
     // to ensure the SQL ingestion worked
     let pool = Pool::new(database_url());
     let sql = "SELECT month_name FROM tesseract_webshop_time;";
```

Comment:
@frabarz wanted to ask about this table in the test. It seems to be hardcoded, without any initialization. I'm not quite sure why the test needed here, seems like it's testing something very specific in a workflow.